istio/restrict-virtual-service-wildcard/restrict-virtual-service-wildcard.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-virtual-service-wildcard
+  annotations:
+    policies.kyverno.io/title: Restrict Virtual Service Host with Wildcards
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.6.2
+    policies.kyverno.io/minversion: 1.6.0
+    kyverno.io/kubernetes-version: "1.23"
+    policies.kyverno.io/subject: VirtualService
+    policies.kyverno.io/description: >-
+      Virtual Services optionally accept a wildcard as an alternative
+      to precise matching. In some cases, this may be too permissive as it
+      would direct unintended traffic to the given resource. This
+      policy enforces that any Virtual Service host does not contain a wildcard
+      character and allows for more governance when a single mesh deployment 
+      model is used.
+spec:
+  validationFailureAction: enforce
```

Comment:
We also try and put (mostly) all validate policies in `audit` mode so they don't have inadvertent consequences for new users.